packages/replay/src/replay.ts

```diff
@@ -217,6 +222,11 @@ export class ReplayContainer implements ReplayContainerInterface {
     return this._options;
   }
 
+  /** Get the replay integrations. [test only] */
+  public getIntegrations(): Record<string, Record<string, unknown>> {
```

Comment:
can we make this private/protected? We can still access it in tests even if it is private, e.g. `replay['_getIntegrations']()` should still work?